bathbot/src/commands/osu/relax/profile.rs

```diff
@@ -0,0 +1,317 @@
+    let builder = MessageBuilder::new()
+        .embed(relax_profile_builder(pagination).unwrap())
+        .attachment("graph.png", graph);
+    orig.create_message(builder).await?;
+
+    Ok(())
+}
+
+pub struct RelaxProfileArgs {
+    user: CachedUser,
+    discord_id: Option<Id<UserMarker>>,
+    info: RelaxPlayersDataResponse,
+    origin: MessageOrigin,
+}
+
+impl RelaxProfileArgs {
+    pub fn new(
+        user: CachedUser,
+        discord_id: Option<Id<UserMarker>>,
+        info: RelaxPlayersDataResponse,
+        origin: MessageOrigin,
+    ) -> Self {
+        Self {
+            user,
+            discord_id,
+            info,
+            origin,
+        }
+    }
+}
+pub fn relax_profile_builder(args: RelaxProfileArgs) -> Result<EmbedBuilder> {
+    let stats = &args.info;
+    let mut description = "__**Relax user statistics".to_string();
+    if let Some(discord_id) = args.discord_id {
+        let _ = write!(description, "for <@{discord_id}>");
+    };
+
+    description.push_str(":**__");
+    let _ = writeln!(
+        description,
+        "\n
+        Accuracy: [`{acc:.2}%`]({origin} \"{acc}\") • \
+        Playcount: `{playcount}`",
+        origin = args.origin,
+        acc = stats.total_accuracy.unwrap_or_default(),
+        playcount = WithComma::new(stats.playcount)
+    );
+    let ss_grades = format!("{}{}", stats.count_ss, grade_emote(Grade::X));
+    let s_grades = format!("{}{}", stats.count_s, grade_emote(Grade::S));
+    let a_grades = format!("{}{}", stats.count_a, grade_emote(Grade::A));
+    let fields = fields![
+        "Count SS", ss_grades, true;
+        "Count S", s_grades, true;
+        "Count A", a_grades, true;
+    ];
+    let embed = EmbedBuilder::new()
+        .author(relax_author_builder(&args))
+        .description(description)
+        .fields(fields)
+        .image(attachment("graph.png"))
+        .thumbnail(args.user.avatar_url.as_ref())
+        .footer(relax_footer_builder(&args));
+    Ok(embed)
+}
+
+fn relax_author_builder(args: &RelaxProfileArgs) -> AuthorBuilder {
+    let country_code = args.user.country_code.as_str();
+    let pp = args.info.total_pp;
+
+    let text = format!(
+        "{name}: {pp}pp (#{rank} {country_code}{country_rank})",
+        name = args.user.username,
+        pp = WithComma::new(pp.unwrap()),
+        rank = args.info.rank.unwrap_or_default(),
+        country_rank = args.info.country_rank.unwrap_or_default(),
+    );
+
+    let url = format!("{RELAX}/users/{}", args.user.user_id);
+    let icon = flag_url(country_code);
+    AuthorBuilder::new(text).url(url).icon_url(icon)
+}
+
+fn relax_footer_builder(args: &RelaxProfileArgs) -> FooterBuilder {
+    let last_update = format!(
+        "Last update: {}",
+        args.info
+            .updated_at
+            .unwrap()
+            .format(NAIVE_DATETIME_FORMAT)
+            .unwrap()
+    );
+    FooterBuilder::new(last_update).icon_url("https://rx.stanr.info/rv-yellowlight-192.png")
```

Comment:
Maybe the icon could be a const and the `top` subcommand could use it as footer icon too?